### PR TITLE
feat(cloning): add export configuration

### PR DESCRIPTION
**Pull request by kgajowy**
As per https://vizzuality.atlassian.net/browse/MARXAN-1109 

This PR adds : 
*  "dummy" pieces handlers
* fix on forever-pending job (`submitted` event instead `finished`)

Unpacked export:
image

### CODE REVIEW

File: api/apps/geoprocessing/src/export/pieces-exporters/project-config.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    await delay();
+
+    const scenarios: { name: string }[] = await this.entityManager.query(
+      `
+       SELECT name FROM scenarios where project_id = $1
+    `,
+      [input.resourceId],
+    );
+
+    const metadata = JSON.stringify({
```
Comment (tiagojsag) [inline api/apps/geoprocessing/src/export/pieces-exporters/project-config.ts:44]: is this meant to mock the project metadata coming from the db?

**Review by tiagojsag — COMMENTED** [review]
I have a couple of question, that are probably not blockers, must just me being new here. feel free to merge if you think it's ok as it is

File: api/apps/geoprocessing/src/export/pieces-exporters/project-config.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  isSupported(piece: ClonePiece): boolean {
+    return piece === ClonePiece.ExportConfig;
+  }
+
+  async run(input: JobInput): Promise<JobOutput> {
+    if (input.resourceKind === ResourceKind.Scenario) {
+      throw new Error(`Exporting scenario is not yet supported.`);
+    }
+
+    await delay();
```
[the remark this answers is not in the document]
Comment (kgajowy) [reply api/apps/geoprocessing/src/export/pieces-exporters/project-config.ts:35]: https://github.com/Vizzuality/marxan-cloud/pull/706/files#diff-21d5c8cc188cdfa256ed298ce1d63d6f66da1968b0ac9ae0e30902f84ca1912cR48-R49

its described in tasks mentioned in takeover plan. 

You are right it is odd - and in final implementation it shouldn't be there at all.

File: api/apps/geoprocessing/src/export/pieces-exporters/project-config.ts (hunk 1 above)
Comment (kgajowy) [reply api/apps/geoprocessing/src/export/pieces-exporters/project-config.ts:44]: it doesn't mock, it should actually be a valid implementation of what's needed for config, according to initial assumptions/HLD.

**Pull request merged**